### DISCUSSION TITLE
Using the fluent TUI command to set drag-law has an error

### DISCUSSION THREAD

**Discussion by toldm** [ansys/pyfluent#1972] [Q&A]
/define/models/dpm/numerics/drag-law
(spherical nonspherical Stokes-Cunningham high-Mach-number Wen-Yu Gidaspow Huilin-Gidaspow Syamlal-OBrien Gibilaro EMMS Filtered Ishii-Zuber Grace)
Drag law ["spherical"] Huilin-Gidaspow

ERROR: chip-exec: function "huilin-gidaspow" not found.
ERROR: chip-exec: function "huilin-gidaspow" not found.
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!! !!!!!!!
!!!!!! Error: DPM drag UDF... !!!!!!!
!!!!!! huilin-gidaspow !!!!!!!
!!!!!! not found -- will not use it. !!!!!!!
!!!!!! !!!!!!!
!!!!!! NOTE: To re-activate the UDF, !!!!!!!
!!!!!! you need to activate the UDF !!!!!!!
!!!!!! in the User Interface again, !!!!!!!
!!!!!! even if it appears active. !!!!!!!
!!!!!! !!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
ERROR: chip-exec: function "huilin-gidaspow" not found.
When I manually select the huilin-gidaspow drag model, it can be set successfully, but when I set it with the TUI command, an error is reported
![image](https://github.com/ansys/pyfluent/assets/132992492/086260c7-4e46-465c-9532-b588785403a4)

**Comment by ansjschuetz** [comment]
To pick a drag law whose name contains capital latters in the TUI, please enclose its name in double quote characters:

/define/models/dpm/numerics/drag-law
(spherical nonspherical Stokes-Cunningham high-Mach-number Wen-Yu Gidaspow Huilin-Gidaspow Syamlal-OBrien Gibilaro EMMS Filtered Ishii-Zuber Grace)
Drag law ["spherical"] **"Huilin-Gidaspow"**
